contribute/CONTRIBUTING-TO-DEVELOPER-DOC.md

```diff
@@ -47,74 +47,5 @@ You can also help with some existing issues under this category available at [de
 - Once the PR is accepted, close the branch.
 - After the PR is merged the development branch in the forked repository can be deleted.
 
-# Git Cheat Sheet 
-
-Fork the openebs/openebs repository into your account, referred in the below instructions as $user. 
-
-## Setting up Development Environment for OpenEBS projects
-```
-working_dir=$GOPATH/src/github.com/openebs
-mkdir -p $working_dir
-cd $working_dir
-```
-
-Set `user` to match your github profile name:
-
-```
-user={your github profile name}
-```
-
-## The steps mentioned here are w.r.t contribution towards openebs/openebs project
-```
-git clone https://github.com/$user/openebs.git
-cd openebs/
-git remote add upstream https://github.com/openebs/openebs.git
-git remote set-url --push upstream no_push
-git remote -v
-```
-
-## Synchronizing your local master(i.e. your $user/openebs project) with upstream master(i.e. openebs/openebs)
```

Comment:
master (

Reply:
That section has already been changed, a review has to be casted on the changed category. That's the right side.